www/src/components/insights/insights.scss

```diff
@@ -0,0 +1,3 @@
+.pgn__data-table-layout-wrapper .pgn__data-table-layout-sidebar {
+  padding: .75rem;
```

Comment:
Curious why we needed to change the default padding for `DataTable` here? We should try to minimize custom CSS styling when using Paragon components as a consumer, as we are here.

Reply:
Actually, I found our standard paddings to small and decided to match padding for the table header.
![image](https://user-images.githubusercontent.com/36768631/175560685-a62f52ca-5cd2-4b2e-817a-e90375793eda.png)
I also went to the Figma and found that it has difference in paddings also.
https://www.figma.com/file/eGmDp94FlqEr4iSqy1Uc1K/Paragon-Design-System?node-id=10469%3A38059
Maybe it is good idea to improve styles for these filter `Card` in the `DataTable`?